addons/web/static/tests/_framework/touch_helpers.js

```diff
@@ -0,0 +1,64 @@
+import { queryFirst } from "@odoo/hoot-dom";
+import { contains } from "./dom_test_helpers";
+
+async function swipe(target, selector, direction, optionsDrag = {}, optionsMoveTo = {}) {
```

Comment:
```suggestion
async function swipe(target, selector, direction) {
```

Remove them as not used

Reply:
https://github.com/odoo/odoo/pull/178014/files#diff-c641eec626dcdfbb1d690724bdf0828843bfaf3dc493c294b9e81596b2016148R224-R239